images/kor/tests/script.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+git clone https://github.com/yonahd/kor.git
+
+# Set up a trap to remove the directory on script exit
+cleanup() {
+    echo "Cleaning up..."
+    rm -rf kor
+    helm uninstall kor -n kor
+    echo "Cleanup complete."
+}
+
+# Register the cleanup function to be called on script exit (EXIT signal)
+trap cleanup EXIT
+
+helm upgrade -i kor \
```

Comment:
nit (non-blocking): all the flags here have been used in the long form except for the `--install` one. Is it fair to change it to the long form too?

```suggestion
helm upgrade --install kor \
```